### DISCUSSION TITLE
mx.logging missing from AIR 33.1?

### DISCUSSION THREAD

**Discussion by misha-cilantro** [airsdk/Adobe-Runtime-Support#829] [Q&A] [answered]
I'm trying to update a very old application to 64-bit for Android. It made use of the mx.logging package from Adobe versions of AIR, but I don't see that package in 33.1. Was it moved somewhere else? Is there info about this (and any other) changes somewhere on Harman's site? Thanks!

**Comment by ajwfrost** [comment, marked as answer]
mx.logging would be part of the Flex SDK, rather than the AIR SDK. So you would need to download the Flex SDK, and then overlay this with the "AIR SDK for Flex developers" in order to get the right combination of tools and libraries..

Simplest may be to use the below link if you don't already have the older SDK available - this should give you the latest Flex SDK.
http://flex.apache.org/installer.html

**Reply by misha-cilantro** [reply]
Ah, yes! I see now, that totally did it. Thanks so much for the quick answer!
